common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -1870,6 +1872,15 @@ object CelebornConf extends Logging {
       .intConf
       .createWithDefault(2)
 
+  val MASTER_SLOT_ASSIGN_MAX_WORKERS: ConfigEntry[Int] =
+    buildConf("celeborn.master.slot.assign.maxWorkers")
+      .categories("master")
+      .version("0.3.1")
+      .doc("Max workers that slots of one shuffle can be allocated on. Will choose the smaller positive one " +
+        s"from Master side and Client side, see `celeborn.client.slot.assign.maxWorkers`.")
```

Comment:
Avoid modifying the `MASTER_SLOT_ASSIGN_MAX_WORKER` doc description after modifying the `CLIENT_SLOT_ASSIGN_MAX_WORKERS` key, should using same as `CLIENT_SLOT_ASSIGN_MAX_WORKERS`.

Like `${CelebornConf. CLIENT_SLOT_ASSIGN_MAX_WORKERS.key}`

Reply:
Thanks @zwangsheng , since `CLIENT_SLOT_ASSIGN_MAX_WORKERS` is defined after `MASTER_SLOT_ASSIGN_MAX_WORKER`, I can't reference `${CelebornConf. CLIENT_SLOT_ASSIGN_MAX_WORKERS.key}` in `MASTER_SLOT_ASSIGN_MAX_WORKER`'s doc, or it would throw error:

```
ConfigurationSuite:
org.apache.celeborn.ConfigurationSuite *** ABORTED ***
  java.lang.RuntimeException: Unable to load a Suite class org.apache.celeborn.ConfigurationSuite that was discovered in the runpath: null
  at org.scalatest.tools.DiscoverySuite$.getSuiteInstance(DiscoverySuite.scala:80)
  at org.scalatest.tools.DiscoverySuite.$anonfun$nestedSuites$1(DiscoverySuite.scala:38)
  at scala.collection.TraversableLike.$anonfun$map$1(TraversableLike.scala:286)
  at scala.collection.Iterator.foreach(Iterator.scala:943)
  at scala.collection.Iterator.foreach$(Iterator.scala:943)
  at scala.collection.AbstractIterator.foreach(Iterator.scala:1431)
  at scala.collection.IterableLike.foreach(IterableLike.scala:74)
  at scala.collection.IterableLike.foreach$(IterableLike.scala:73)
  at scala.collection.AbstractIterable.foreach(Iterable.scala:56)
  at scala.collection.TraversableLike.map(TraversableLike.scala:286)
  ...
  Cause: java.lang.ExceptionInInitializerError:
  at org.apache.celeborn.ConfigurationSuite.<init>(ConfigurationSuite.scala:49)
  at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
  at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
  at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
  at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
  at java.lang.Class.newInstance(Class.java:442)
  at org.scalatest.tools.DiscoverySuite$.getSuiteInstance(DiscoverySuite.scala:66)
  at org.scalatest.tools.DiscoverySuite.$anonfun$nestedSuites$1(DiscoverySuite.scala:38)
  at scala.collection.TraversableLike.$anonfun$map$1(TraversableLike.scala:286)
  at scala.collection.Iterator.foreach(Iterator.scala:943)
  ...
  Cause: java.lang.NullPointerException:
  at org.apache.celeborn.common.CelebornConf$.<init>(CelebornConf.scala:1880)
  at org.apache.celeborn.common.CelebornConf$.<clinit>(CelebornConf.scala)
  at org.apache.celeborn.ConfigurationSuite.<init>(ConfigurationSuite.scala:49)
  at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
  at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
  at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
  at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
  at java.lang.Class.newInstance(Class.java:442)
  at org.scalatest.tools.DiscoverySuite$.getSuiteInstance(DiscoverySuite.scala:66)
  at org.scalatest.tools.DiscoverySuite.$anonfun$nestedSuites$1(DiscoverySuite.scala:38)
  ...
*** RUN ABORTED ***
  java.lang.ExceptionInInitializerError:
  at org.apache.celeborn.ConfigurationSuite.<init>(ConfigurationSuite.scala:49)
  at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
  at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
  at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
  at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
  at java.lang.Class.newInstance(Class.java:442)
  at org.scalatest.tools.DiscoverySuite$.getSuiteInstance(DiscoverySuite.scala:66)
  at org.scalatest.tools.DiscoverySuite.$anonfun$nestedSuites$1(DiscoverySuite.scala:38)
  at scala.collection.TraversableLike.$anonfun$map$1(TraversableLike.scala:286)
  at scala.collection.Iterator.foreach(Iterator.scala:943)
  ...
  Cause: java.lang.NullPointerException:
  at org.apache.celeborn.common.CelebornConf$.<init>(CelebornConf.scala:1880)
  at org.apache.celeborn.common.CelebornConf$.<clinit>(CelebornConf.scala)
  at org.apache.celeborn.ConfigurationSuite.<init>(ConfigurationSuite.scala:49)
  at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
  at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
  at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
  at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
  at java.lang.Class.newInstance(Class.java:442)
  at org.scalatest.tools.DiscoverySuite$.getSuiteInstance(DiscoverySuite.scala:66)
  at org.scalatest.tools.DiscoverySuite.$anonfun$nestedSuites$1(DiscoverySuite.scala:38)
  ...
```